src/NzbDrone.Api/Extensions/Pipelines/CacheHeaderPipeline.cs

```diff
@@ -15,7 +15,7 @@ public CacheHeaderPipeline(ICacheableSpecification cacheableSpecification)
 
         public void Register(IPipelines pipelines)
         {
-            pipelines.AfterRequest.AddItemToStartOfPipeline(Handle);
+            pipelines.AfterRequest.AddItemToStartOfPipeline(Handle, false);
```

Comment:
For some reason this must be explicitly stated: `Whether to replace an existing item with the same name in its current place, rather than at the position requested. Defaults to false.`
http://www.nudoq.org/Snapshots/Packages/Nancy/Nancy/AsyncNamedPipelineBase(TAsyncDelegate,TSyncDelegate)/M/AddItemToEndOfPipeline/